spec/lib/mailinator/email_spec.rb

```diff
@@ -52,6 +50,7 @@
     let(:response) { subject.delete('abcd1234') }
 
     it 'deletes an email' do
+      response = Mailinator::Email.delete('abcd1234')
```

Comment:
This should be covered by the `let(:response) { subject.delete('abcd1234') }` on the lines above.